syncthing-client/src/main/kotlin/net/syncthing/java/client/SyncthingClient.kt

```diff
@@ -35,6 +35,10 @@ import java.util.concurrent.Executors
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.atomic.AtomicBoolean
 import kotlin.collections.ArrayList
+import kotlin.collections.HashMap
+import kotlin.collections.component1
+import kotlin.collections.component2
+import kotlin.collections.set
```

Comment:
Looks like you imported different collections again. Maybe something is misconfigured in your Android Studio?